src/status/CountryChart.js

```diff
@@ -44,6 +44,19 @@ const CountryChart = ({ t, peerLocations, className }) => {
     legend: {
       display: true,
       position: 'bottom'
+    },
+    tooltips: {
+      displayColors: false,
+      callbacks: {
+        title: (tooltipItem, data) => data.labels[tooltipItem[0].index],
+        label: (tooltipItem, data) => {
+          const percent = data['datasets'][0]['data'][tooltipItem['index']]
+          const count = Math.round((percent * totalCountries) / 100)
+          const peerStr = t('peersCount', { count: count })
+
+          return `${percent}% (${count} ${peerStr})`
```

Comment:
I'd translate this whole string as it wouldn't make sense in a right-to-left language.

Reply:
![rollsafe](https://user-images.githubusercontent.com/33324750/47210090-e58cf280-d389-11e8-9b94-bf34155476f8.jpg)